### PR TITLE
New working clones

### PR DESCRIPTION
**Pull request by clawgrip**
------------------
Hot Blocks - Tetrix II (set 2) [Victor Fernandez (City Game), ClawGrip]
Hot Blocks - Tetrix II (set 3) [jordigahan, ClawGrip]

### CODE REVIEW

File: src/mame/drivers/hotblock.cpp [hunk 1]
```diff
@@ -176,19 +188,18 @@ static INPUT_PORTS_START( hotblock )
 	PORT_BIT( 0x80, IP_ACTIVE_LOW, IPT_BUTTON2 ) PORT_PLAYER(2) // used to get test mode
 INPUT_PORTS_END
 
-
 void hotblock_state::hotblock(machine_config &config)
 {
-	/* basic machine hardware */
-	I8088(config, m_maincpu, 24_MHz_XTAL / 3); // Unknown clock
```
Comment (Osso13) [inline src/mame/drivers/hotblock.cpp:183]: The PCB info that was at the top of the driver reported a P8088-1 as the CPU.
I'm not sure completely removing this info from the driver is the best course of action.
Same goes for changing the 24C04 to 24C02 and the YM2149 to AY8910.
Isn't it possible this game ran on two different PCB revision with different (mostly compatible) components? If yes, shouldn't this be documented?

Comment (clawgrip) [reply src/mame/drivers/hotblock.cpp:183]: Yes, I asked @DavidHaywood about it and he did not remember having a PCB pic or something. I've located five different PCBs and all of them had a NEC V20.
About the 24C04, it was probably used on some boards, but wasting half the size, as it's just using 0x0100 bits, so I'd say using a 24C02 is the right choice...
Anyway, you're right, this changes should be documented. I'll add another commit on a few minutes.
Thx!

Comment (Osso13) [reply src/mame/drivers/hotblock.cpp:183]: Thanks, I believe this is a pic of the dumped PCB, but it was done in the mid 2000 and the definition isn't good enough:

https://imgur.com/a/ISVU2tD

Comment (clawgrip) [reply src/mame/drivers/hotblock.cpp:183]: I'd say the CPU has its surface scratched out...

**Pull request merged**